### ISSUE TITLE
make copyright year dynamic

### ISSUE THREAD

**Issue by jonotron** [TwoStoryRobot/twostoryrobot.com#121]
The copyright year at the bottom of the page is static to 2018, it should be updated to be dynamic to the current year.

**Comment by jonotron** [comment]
The blog pages are a good example of this done.

**Comment by chrisfosterelli** [comment]
From a marketing perspective, it's nice having an up-to-date number on the site. From a legal perspective, IANAL but I did some reading on this a while ago. Some notes if it's interesting:

The year is supposed to represent the date the work _was first published_. So technically it could be something like "© Two Story Robot 2018". For sites like blogs that might have content first published across a span of years, you can use a range like "© Two Story Robot 2018 - 2020". Apparently including only the most recent date is considered problematic because someone could create some similar content after yours, but if it looks like you didn't claim copyright until after their work was created, then they could claim to have the original copyright on it. 

Because some people find it difficult to sort through the first publish dates, they just opt to not include the year at all (e.g. "© Two Story Robot"). This is what I did for my site as it looks a little cleaner and I don't care too much. This is technically not a valid copyright notice, but in practice it's fine because of the next point.

Additionally, the copyright notice is not technically required at all. By default, copyright is assigned for a period of the author's life + 50 years immediately on publication. If we were citing someone else's work, it'd be important to highlight the copyright belongs to them, but since it's the company site it is implied. It is primarily considered to be informational for someone who might be trying to find the copyright holder of a work to request a license. That said, it has been used in Canada as proof that an infringing party should have known that work was copyright so there is arguably some benefit.

**TL;DR:** If going to indicate a year, the best option is to include the initial year *and* the current year as a range, but some people have advocated for removing the year altogether.

https://www.copyrightlaws.com/copyright-symbol-notice-year/

**Comment by jonotron** [comment]
Thanks for the overview here @chrisfosterelli! Let's go with the year span, 2018-Current year.
